onnx/backend/test/case/node/averagepool.py

```diff
@@ -197,11 +198,12 @@ def export_averagepool_1d_default() -> None:
         )
         x = np.random.randn(1, 3, 32).astype(np.float32)
         x_shape = np.shape(x)
+        dilations = [1]
         kernel_shape = [2]
         strides = [1]
         out_shape = get_output_shape("VALID", x_shape[2:], kernel_shape, strides)
         padded = x
-        y = pool(padded, x_shape, kernel_shape, strides, out_shape, [0], "AVG")
+        y = pool(padded, x_shape, kernel_shape, strides, out_shape, [0, 0], "AVG", dilations=dilations)
```

Comment:
Can we not handle this (and similar examples below) by using a default-value of None for dilations in the implementation of `pool`?

Reply:
That would match the ONNX representation above (where `dilations` is not specified).

Reply:
dilations as an argument removed from the call